wgengine/magicsock/magicsock.go

```diff
@@ -4040,6 +4069,11 @@ func ippDebugString(ua netip.AddrPort) string {
 // recalculated.
 type endpointSendFunc func([][]byte) error
 
+type endpointDisco struct {
+	key   key.DiscoPublic // for discovery messages.
+	short string          // ShortString of discoKey.
+}
```

Comment:
document that this is immutable? I hope/assume.

Reply:
It is. Will do.

Reply:
https://github.com/tailscale/tailscale/pull/7833